asapdiscovery-genetics/asapdiscovery/genetics/scripts/structure_alignment.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+from pathlib import Path
+
+from asapdiscovery.genetics.calculate_rmsd import (
+    save_alignment_pymol,
+    select_best_colabfold,
+)
+
+parser = argparse.ArgumentParser(
+    description="Align PDB structures generated from ColabFold"
+)
+
+parser.add_argument(
+    "--seq-file",
+    type=str,
+    required=True,
+    help="Path to the csv with the sequences.",
+)
+parser.add_argument(
+    "--ref-pdb",
+    type=str,
+    required=True,
+    help="Path to PDB file of reference structure.",
+)
+
+parser.add_argument(
+    "--save-dir",
+    type=str,
+    required=False,
+    default="aligned_pdbs/",
+    help="Path directory to save PDB of aligned target protein",
+)
+
+parser.add_argument(
+    "--results-dir",
+    type=str,
+    required=False,
+    default="./",
+    help="Path to folder where all ColabFold results are stored.",
+)
+
+parser.add_argument(
+    "--out-dir",
+    type=str,
+    required=False,
+    default="./",
+    help="Path to output folder given on ColabFold run.",
+)
+
+parser.add_argument(
+    "--pymol-save",
+    type=str,
+    required=False,
+    default="aligned_proteins.pse",
+    help="Path to save pymol session with aligned proteins",
+)
+
+
+def main():
+    args = parser.parse_args()
+    # check all the required files exist
+    seq_file = Path(args.seq_file)
+    if not seq_file.exists():
+        raise FileNotFoundError(f"Sequence file {seq_file} does not exist")
+    query_name = seq_file.stem
+
+    ref_pdb = Path(args.ref_pdb)
+    if not ref_pdb.exists():
+        raise FileNotFoundError(f"Ref PDB file {ref_pdb} does not exist")
+
+    results_dir = Path(args.results_dir)
+    if not results_dir.exists():
+        raise FileNotFoundError(
+            f"The folder with ColabFold results {results_dir} does not exist"
+        )
+    save_dir = Path(args.save_dir)
+    save_dir.mkdir(parents=True, exist_ok=True)
+
+    pymol_save = args.pymol_save
+
+    aligned_pdbs = []
+    seq_labels = []
+    with open(seq_file) as f:
```

Comment:
Should read a csv with something like `pandas.read_csv()` rather than by hand for more consistency and portability. This will handle everything for you. You can then just iterate over the resulting `DataFrame`